src/common/hooks/useOrderLineLocationsByTenants/useOrderLineLocationsByTenants.js

```diff
@@ -0,0 +1,52 @@
+import {
+  useCentralOrderingContext,
+  useTenantHoldingsAndLocations,
+  useLocationsQuery,
+  useReceivingTenantIdsAndLocations,
+} from '@folio/stripes-acq-components';
+import { useStripes } from '@folio/stripes/core';
+
+const DEFAULT_DATA = [];
+
+export const useOrderLineLocationsByTenants = ({ receivingTenantIdsByLocations = DEFAULT_DATA, instanceId }) => {
```

Comment:
What does the order line have to do with it, if the hook doesn't have references to the line?